### PR TITLE
Fix/fix get all pets return type

### PR DESCRIPTION
**Pull request by marclll**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5247402&bid=17197184)

File: src/test/java/org/pesmypetcare/webservice/controller/appmanager/MyPetCareRestControllerTest.java [hunk 1]
```diff
@@ -54,7 +52,7 @@ public void signUpShouldReturnStatusOK() throws Exception {
             .content(jsonUser1).param(key, password))
             .andExpect(status().isOk());
     }
-
+/*
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/controller/appmanager/MyPetCareRestControllerTest.java]: Codacy Issue found: [Block comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 47.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5247402)

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by enrichp — APPROVED** [review]
Dan un poco de pena

File: src/main/java/org/pesmypetcare/webservice/dao/PetDaoImpl.java [hunk 2]
```diff
@@ -79,19 +80,22 @@ public PetEntity getPetData(String owner, String name) throws DatabaseAccessExce
     }
 
     @Override
-    public Map<String, PetEntity> getAllPetsData(String owner) throws DatabaseAccessException {
+    public List<Map<String, Object>> getAllPetsData(String owner) throws DatabaseAccessException {
         CollectionReference petsRef = usersRef.document(owner).collection(PETS_KEY);
-        Map<String, PetEntity> mapa = new HashMap<>();
+        List<Map<String, Object>> listaExterna = new ArrayList<>();
```
Comment (AlbertPG) [inline src/main/java/org/pesmypetcare/webservice/dao/PetDaoImpl.java]: There are some variables that are not written in english

**Review by AlbertPG — CHANGES_REQUESTED** [review]
Take a look at this

**Review by AlbertPG — APPROVED** [review]
Okey

**Review by santidrj — COMMENTED** [review]
👎 😈

**Review by santidrj — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
Okey

**Review by enrichp — APPROVED** [review]
Pesao

**Review by enrichp — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
Okey

**Review by santidrj — APPROVED** [review]
😪

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by AlvarTB — APPROVED** [review]

**Review by AlvarTB — COMMENTED** [review]
k

**Pull request merged**